RoomsEyeView/Logic/vtkSlicerRoomsEyeViewModuleLogic.cxx

```diff
@@ -1645,3 +1866,167 @@ std::string vtkSlicerRoomsEyeViewModuleLogic::GetStateForPartType(std::string pa
 
   return stateStr;
 }
+
+//-----------------------------------------------------------------------------
+bool vtkSlicerRoomsEyeViewModuleLogic::GetTransformNodeBetween(vtkSlicerIECTransformLogic::CoordinateSystemIdentifier fromFrame, vtkSlicerIECTransformLogic::CoordinateSystemIdentifier toFrame,

```

Comment:
Is it possible that you have this function more than once? I see a `beamsLogic->GetTransformNodeBetween` call a bit above, and now the same function is defined here. Keep only the one in the lower level class if they are duplicated.